### DISCUSSION TITLE
How to contributing?

### DISCUSSION THREAD

**Discussion by bickelmeister** [papsign/Ktor-OpenAPI-Generator#133] [General]
Is it wanted to contribute to the further development? If so, are there rules or guidance on how best to get involved?

**Comment by Wicpar** [comment]
Thank you for your proposal, it is most welcome as i do not work with ktor anymore.

Currently the best way is to pull request a feature with relevant unit tests. I review the code and if it isn't breaking and works it is usually accepted.
I also accept refactoring if it has a tangible advantage for reusability and code clarity.
